### PR TITLE
Design markup changes 

### PR DESCRIPTION
**Pull request by xswoodsx**
Changes made following chat with Dom, listed as below. 

- Back links on all pages
- Update the test data
- Add search page at end of flow with pending tag on corfe hills
- fix filter on tags 
- partnership status filter header too big
- Pass the name through to success on single form 
- error summaries to top of page 
- MAT name on school result 
- Distance boxes on one line in filter partial 

NOTE - Mine and Ethans linters didn't agree on how to format the JS functions, that's the main reason for the number of additions/deletions

### CODE REVIEW

File: app/views/declare-partnership/single/failed.html [hunk 1]
```diff
@@ -34,6 +27,13 @@ <h1 class="govuk-heading-xl">
                     }
                 ]
             }) }}
+            <h1 class="govuk-heading-xl">
+                Add new partnerships
+            </h1>
+            {{ govukWarningText({
+                            text: "You should only add a school to your list if they have signed a partnership contract with you",
+                            iconFallbackText: "Warning"
+                        }) }}
```
Comment (kicferk1) [inline app/views/declare-partnership/single/failed.html]: Whitespace isn't quite right.

Also, are we sure about moving the page header below error summary?

File: app/assets/javascripts/school-search/school-search.js [hunk 2]
```diff
@@ -1,78 +1,114 @@
...
+  const resultsHtml = window.nunjucks.render('school-results.html', {
+    schools: filteredSchools(),
+  });
+  document.getElementById('school-results').innerHTML = resultsHtml;
 }
 
-function filteredSchools () {
-  let schools = allSchools
+function filteredSchools() {
+  let schools = getAllSchools();
```
Comment (kicferk1) [inline app/assets/javascripts/school-search/school-search.js:26]: Probably worth comparing linter setup with Ethan, picking one, and adding a note about it in the readme. 

You could even do something like https://prettier.io/docs/en/precommit.html to include make your linter (article uses `prettier`) format everything each time you commit changes.

**Review by kicferk1 — COMMENTED** [review]
Some merge conflicts :(

File: app/views/declare-partnership/single/failed.html (hunk 1 above)
Comment (ethanmills) [reply app/views/declare-partnership/single/failed.html]: The error summary thing was a specific request from Dom

File: app/assets/javascripts/school-search/school-search.js (hunk 2 above)
Comment (ethanmills) [reply app/assets/javascripts/school-search/school-search.js:26]: Yeah, we noticed this. I'm adding eslint into the project to define shared settings

File: app/views/declare-partnership/single/failed.html (hunk 1 above)
Comment (xswoodsx) [reply app/views/declare-partnership/single/failed.html]: Ah yeah, good spot on the whitesapce. 

We had a chat with Dom yesterday about the error summary and she asked to put the page header below the error summary. There's a screenshot on the miro board how the coronavirus service has it.

Comment (kicferk1) [reply app/views/declare-partnership/single/failed.html]: Ah, alright, I should have read https://design-system.service.gov.uk/components/error-summary/ in more detail.
> Put the error summary at the top of the main container. If your page includes breadcrumbs or a back link, place it below these, but above the \<h1>.

**Review by kicferk1 — APPROVED** [review]

**Pull request merged**